e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -40,5 +36,29 @@ Scenario: Re-enrolling an agent
 @revoke-token
 Scenario: Revoking the enrollment token for an agent
   Given an agent is deployed to Fleet
+    And the agent is un-enrolled
+    And the "agent" process is "stopped" on the host
   When the enrollment token is revoked
-  Then an attempt to enroll a new agent fails
+  Then an attempt to enroll an agent with the old token fails
+
+@package-added-to-default-config
+Scenario: Adding packages to configurations
```

Comment:
we've decided that the default policy ( configuration ) is the same as any newly created configuration so this test is not needed in that angle, if we do want to try to set up individual Integrations, we can certainly do that - but would be better to file them to their own individual file(s).  I'd say we can leave this out for now.